Evelin-Gongora/Tarea2-HTML/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="es">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>París</title>
+  <link rel="stylesheet" href="css/styles.css">
+  <link rel="icon" href="resources/tower.png">
+</head>
+<body>
+  <main>
+    <article>
+      <section>
+        <div class="container">
+          <div class="container-child">
+            <hgroup>
+              <div class="container-header">
+                <h2 class="phrase">THE FASHION ISSUE - GUEST EDITOR CHIARA SCELSI</h2>
+                <h1 class="title">L'OFFICIEL</h1>
+              </div>
+              <div class="container-italia">
+                <h3 class="italia">ITALIA</h3>
+              </div>
+            </hgroup>
+            <div class="container-date">
+              <h4 class="date">N° 32 - FEBBRAIO 2020</h4>
+            </div>
+            <div class="container-editor">
+              <h4 class="editor">DE LA COUTURE ET DE LA MODE DE PARIS</h4>
+            </div>
+            <strong>
```

Comment:
@evelin0521 , <strong> tag is suited to make texts font bolder not to contain other tags